### ISSUE TITLE
Start finetuning from last epoch

### ISSUE THREAD

**Issue by vaishnavipatil29** [Open-Speech-EkStep/vakyansh-wav2vec2-experimentation#22]
Is there a way to restart finetuning of the model from the best checkpoint or last checkpoint where it had stopped before?

**Comment by harveenchadha** [comment]
Yes, please rename the checkpoint as checkpoint_last.pt and place it under the respective checkpoint directory.
